frontend/components/MostViewed.tsx

```diff
@@ -46,30 +47,21 @@ interface Trail {
     linkText: string;
 }
 
-export default class MostViewed extends Component<{}, { trails: Trail[] }> {
-    constructor(props: {}) {
-        super(props);
-        this.state = {
-            trails: [],
-        };
-    }
-
-    public componentDidMount() {
-        fetch('https://api.nextgen.guardianapps.co.uk/most-read-geo.json?guui')
-            .then(resp => resp.json())
-            .then(({ trails }) => {
-                this.setState({
-                    trails,
-                });
-            });
-    }
+const fetchTrails = async () => {
+    const resp = await fetch(
```

Comment:
Are we happy to introduce `async`/`await` now? Does this comment https://github.com/guardian/frontend/pull/16864#issuecomment-303457881 no longer stand?

Reply:
Hadn't realised. 

There's a babel plugin that translates them to promise chains: 
https://github.com/rpetrich/babel-plugin-transform-async-to-promises

Reply:
Ok this is a doozy
we also have: 
- https://github.com/babel/babel/pull/7076
- https://github.com/MatAtBread/nodent

So there's a lot of complexity and drama about making this work. I'll pull this out to a promise chain, but would like to spike this at some point to see what we can do.